frontend/packages/noobaa-storage-plugin/src/utils.ts

```diff
@@ -6,7 +6,7 @@ import { K8sResourceKind } from '@console/internal/module/k8s';
 export const filterNooBaaAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'NooBaa');
 
-export const getGaugeValue = (data) => _.get(data, 'data.result[0].value[1]', null);
+export const getGaugeValue = (data) => _.get(data, 'data.result[0].value[1]');
```

Comment:
Send it with refactor, also FYI we will be moving this too console shared so it can be used in both the dashboards

Reply:
It won't. I had looked into it. `null` guards are used in places where `getGaugeValue` is used. They won't break when `null` is replaced by `undefined`. 
Thanks but I don't see any merge conflicts as of now.

Reply:
> Send it with refactor, also FYI we will be moving this to console shared so it can be used in both the dashboards

Merge conflicts are not happening as of now.

Reply:
Yes I remember, I added `null` guards to remove it later.
Thanks for clarifying.